shroud/main.py

```diff
@@ -406,7 +407,7 @@ def main_with_args(args):
 
     for filename in args.filename:
         ext = os.path.splitext(filename)[1]
-        if ext == ".yaml":
+        if re.search(ext, "[.](yaml|yml)$"):
```

Comment:
I would suggest a simpler solution: ``ext in [".yaml", ".yml"]``. This avoids requiring another module (re).

Reply:
re is standard in the Python library - are you sure?

Reply:
ok should be updated. happy to do it :)

Reply:
It avoids an import even if it's in the standard library and I think it's easier to read.

Reply:
Totally agree!